incubator/starter/README.md

```diff
@@ -1,8 +1,50 @@
 # Starter Stack
 
-A minimal, runnable starter stack that can be used as a base for developing new stacks. This stack provides a sample application template consisting of single line bash scripts, along with all the settings in Dockerfiles and manifests to allow you to use the Appsody CLI to test out init/run/debug/test/build/deploy ahead of starting to make changes to a copy of this stack for yourself.
+A minimal, runnable starter stack that can be used as a base for developing new stacks. This stack provides a sample application template consisting of single line bash scripts, along with all the settings in Dockerfiles and manifests to allow you to use the Appsody CLI to test out package/init/run/debug/test/build/deploy ahead of starting to make changes to a copy of this stack for yourself.
 
-For any new stack, this Readme should explain in detail about this stack and any templates included, e.g.:
+To make a copy of this stack as a basis for your new stack, enter, for example:
```

Comment:
Might be good to explain this in the way the help text is written. Something like.. 

To create a new stack called `my-stack` based on this sample starter stack.

Reply:
Agreed